### PR TITLE
Migrate to DVCLive v2

### PR DESCRIPTION
**Pull request by mattseddon**
Changes required to migrate to v2 of DVCLive

### CODE REVIEW

**Review by sroy3 — APPROVED** [review]

**Review by shcheklein — APPROVED** [review]

File: dvc.yaml [hunk 1]
```diff
@@ -26,7 +26,6 @@ plots:
       y: digit
       template: bar_horizontal
       title: Histogram of Predictions
-  - training/plots/images
```
Comment (dberenbaum) [inline dvc.yaml:29]: Any reason this was dropped?

Comment (mattseddon) [reply dvc.yaml:29]: mistake

Comment (mattseddon) [reply dvc.yaml:29]: fixed here: https://github.com/iterative/vscode-dvc-demo/pull/42

Comment (mattseddon) [reply dvc.yaml:29]: @dberenbaum please LMK (when you have time) if there are any further changes that need to get made to remove legacy syntax 🙏🏻

**Pull request merged**